test/runtests.jl

```diff
@@ -1,207 +1,385 @@
+        cuba, polyroots = results
+        @test length(filter(p -> p.reachable, results)) == 2
+        @test length(filter(p -> p.runtests, results)) == 2
+        @test cuba.cirrus
+        @test !polyroots.docs # Documentation is in the README!
+        # We can also use broadcasting!
+        fields = [(p.uuid, p.tree_hash) for p in results]
+        fields2 = [(p.uuid, p.tree_hash) for p in analyze.(packages; auth)]
+        @test issetequal(fields, fields2)
+
+        # Test `analyze` with `root` argument specified
+        mktempdir() do root
+            @test isempty(readdir(root)) # dir starts empty
+            measurements2 = analyze(find_package("Measurements"; version=:dev); auth, root)
+            @test !isempty(readdir(root)) # code gets downloaded there
+            @test measurements2.version === nothing
+
+            measurements3 = analyze(find_package("Measurements"); auth, root)
+            @test isequal(measurements, measurements3) # same version, just with a root
+            @test measurements.version isa VersionNumber
+
+            measurements4 = analyze(find_package("Measurements"; version=v"2.8.0"); auth, root)
+            @test measurements4.version == v"2.8.0"
+            @test isdir(joinpath(root, "Measurements", "PwGjt")) # not cleaned up yet
+        end
     end
-end
 
-@testset "`analyze`" begin
-    for pkg in (analyze(PackageAnalyzer; auth), analyze("https://github.com/giordano/PackageAnalyzer.jl"; auth), analyze(joinpath(@__DIR__, ".."); auth))
-        @test pkg.uuid == UUID("e713c705-17e4-4cec-abe0-95bf5bf3e10c")
-        @test pkg.reachable == true # default
-        @test pkg.docs == true
-        @test pkg.runtests == true # here we are!
-        @test pkg.github_actions == true
-        @test length(pkg.license_files) == 1
-        @test pkg.license_files[1].licenses_found == ["MIT"]
-        @test pkg.license_files[1].license_filename == "LICENSE"
-        @test pkg.license_files[1].license_file_percent_covered > 90
-        @test pkg.license_files isa Vector{<:NamedTuple}
-        @test keys(pkg.license_files[1]) == (:license_filename, :licenses_found, :license_file_percent_covered)
-        @test isempty(pkg.licenses_in_project)
+    @testset "find_packages_in_manifest" begin
+        pkgs = find_packages_in_manifest(joinpath(pkgdir(PackageAnalyzer), "Manifest.toml"))
+        @test pkgs isa Vector{PkgSource}
+        few = first(pkgs, 3)
+        # Should be all release deps here
+        @test all(p -> p isa Release, few)
+        results = analyze_packages(few; auth)
+        @test length(results) == length(few)
+        # Version number saved out
+        @test all(results[i].version == few[i].version for i in 1:length(few))
+    end
+
+    @testset "`analyze`" begin
+        # `dev` analysis:
+        for pkg in (analyze(PackageAnalyzer; auth), analyze("https://github.com/giordano/PackageAnalyzer.jl"; auth), analyze(joinpath(@__DIR__, ".."); auth))
```

Comment:
I guess we can update the URL now :sweat_smile: 
```suggestion
        for pkg in (analyze(PackageAnalyzer; auth), analyze("https://github.com/JuliaEcosystem/PackageAnalyzer.jl"; auth), analyze(joinpath(@__DIR__, ".."); auth))
```